packages/bits/demo/src/components/demo/combobox-v2/combobox-v2-virtual-scroll/combobox-v2-virtual-scroll.example.component.ts

```diff
@@ -39,23 +37,12 @@ export class ComboboxV2VirtualScrollExampleComponent implements OnDestroy, After
         });
 
         this.combobox.valueChanged.pipe(
-            tap(v => this.filteredItems = of(this.filterItems(v as string))),
```

Comment:
@DmytroPutrin, I'm not sure, but I think this is needed to filter out items that don't match the string currently entered into the combobox.

Reply:
But, maybe this is what you said Dima G. found a bug about, and this is part of the fix?

Reply:
As long as typeahead successfully filters out the non-matching strings, I think we're good.